common/ManagementTestShared/Redesign/ManagementRecordedTestBase.cs

```diff
@@ -169,7 +169,8 @@ private ArmEnvironment GetArmEnvironment(TestEnvironment environment)
             if (baseUri == ArmEnvironment.AzureGovernment.Endpoint)
                 return ArmEnvironment.AzureGovernment;
 
-            return new ArmEnvironment(new Uri(endpoint), SessionEnvironment.ServiceManagementUrl ?? $"{endpoint}/.default");
+            //return new ArmEnvironment(new Uri(endpoint), SessionEnvironment.ServiceManagementUrl ?? $"{endpoint}/.default");
+            return new ArmEnvironment(new Uri(endpoint), environment.ServiceManagementUrl ?? $"{endpoint}/.default");
```

Comment:
This is used for all mgmt sdk tests. Please roll back the change, and use other ways to get `ServiceManagementUrl`.

Reply:
I've used this to be able to run the test in DF environment and I pushed this change accidentally. rolled back, thanks.